client/extensions/woocommerce/components/sparkline/test/index.jsx

```diff
@@ -13,61 +13,73 @@ import React from 'react';
 import Sparkline from '../index';
 
 describe( 'Sparkline', () => {
+	const shallowWithoutLifecycle = arg => shallow( arg, { disableLifecycleMethods: true } );
```

Comment:
Nice 👍

Reply:
`disableLifeycycleMethods: true` used to be the default.  Most of the changes between 2-->3 seem like they are removing automated mocking-ish behavior to try and make functions behave more realistically.